### PR TITLE
Fixes #101 - Add performance page and styles

### LINKED ISSUE #101: HTML/CSS - Performance - est. 1 day

### PR DESCRIPTION
**Pull request by acacheung**
Updates Maine lettering

### CODE REVIEW

**Comment by acacheung** [comment]
@Ctannerweb :eyes:

**Comment by Ctannerweb** [comment]
Should shared.css be heading.css? Not sure how I feel about shared.css, defeats the purpose of keeping things modular, feels like the knick knack chore :P

Thoughts? @acacheung

File: 2016/styles/type.css [hunk 1]
```diff
@@ -13,9 +13,17 @@
...
   clip: rect(1px, 1px, 1px, 1px);
 }
 
 .t-link {
   text-decoration: none;
 }
+
+.t-center {
+  text-align: center;
+}
```
Comment (Ctannerweb) [inline 2016/styles/type.css]: I'd rather add centering with css over a class name. The more single style class names we add the more times the browser has to go back to the .css to find the a classes styling. Thoughts?

File: 2016/styles/modules/performance.css [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+    margin-bottom: 130px;
+  }
+  a {
+    text-decoration: none;
+  }
+}
+
+.performance__social {
+  margin-right: 10px;
+  margin-left: 10px;
```
Comment (Ctannerweb) [inline 2016/styles/modules/performance.css]: Remove margin-right and left. In relation to clickable area problem

File: 2016/styles/modules/performance.css [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+.performance__socials {
+  text-align: center;
+  @media (max-width: 939px) {
+    margin-bottom: 30px;
+  }
+  @media (min-width: 940px) {
+    margin-bottom: 130px;
+  }
+  a {
+    text-decoration: none;
```
Comment (Ctannerweb) [inline 2016/styles/modules/performance.css]: add padding-right: 10px padding-left: 10px display: inline-block to fix clickable area problem on the social icons

File: 2016/styles/modules/shared.css [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+      width: 28px;
+    }
+    @media (min-width: 940px) {
+      margin-bottom: 24px;
+      height: 50px;
+      width: 50px;
+    }
+  }
+
+  .team & {
```
Comment (Ctannerweb) [inline 2016/styles/modules/shared.css]: change to .l-team

File: maine/team.html [hunk 5]
```diff
@@ -54,7 +50,7 @@
         <a class="nav__link spot-color" href="attend.html">Attend</a>
       </nav>
     </header>
-    <main class="main team">
+    <main class="main l-team t-center team">
```
Comment (Ctannerweb) [inline maine/team.html]: Remove team class. No styles affiliated with the class name.

**Comment by Ctannerweb** [comment]
Do we want mobile view to come in at 940px site wide? I have 800px but can easily change :)

**Pull request merged**